app/src/main/java/org/simple/clinic/activity/placeholder/PlaceholderScreen.kt

```diff
@@ -5,15 +5,22 @@ import android.util.AttributeSet
 import android.widget.RelativeLayout
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
-import kotlinx.android.synthetic.main.screen_placeholder.view.*
 import org.simple.clinic.await.Await
 import org.simple.clinic.await.Checkpoint
+import org.simple.clinic.databinding.ScreenPlaceholderBinding
 import java.util.concurrent.TimeUnit.SECONDS
 
 class PlaceholderScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
   private val delayToShowMessage = SECONDS.toMillis(3).toInt()
   private val await = Await(listOf(Checkpoint.unit(delayToShowMessage)))
   private var awaitDisposable: Disposable? = null
+  private val binding: ScreenPlaceholderBinding? = null
+
+  private val loadingTextLayout
+  get() = binding!!.loadingTextLayout
+
+  private val loadingProgressBar
+  get() = binding!!.loadingProgressBar
```

Comment:
These are indented incorrectly, the `get() = ` statements must be indented at one more level than the property declaration.

Have you imported the code styles as mentioned in the [DOCS](https://github.com/simpledotorg/simple-android#code-styles)? Please import and run the IDE autoformatting before you commit anything.